src/main/scripts/setup-glassfish.py

```diff
@@ -1,14 +1,12 @@
 #!/usr/bin/env python
+
 from __future__ import print_function
 
 import threading
 import platform
 import shlex
 import subprocess
-try:
-    import io
-except ImportError:
-    import StringIO as io
+import io
```

Comment:
Should this be reverted back to the try except statement since I guess that made this import work on 2 & 3?

Reply:
The `io` module is in Python 2.6+. I've tested this with Python 2.7 and 3.6.

Reply:
Ah fair enough, just thought I'd check!